src/libdredd/src/mutate_ast_consumer.cc

```diff
@@ -194,7 +194,7 @@ std::string MutateAstConsumer::GetRegularDreddPreludeCpp(
   // this mutant. Then `local_value % 64` determines which bit of that element
   // needs to be set in order to enable the mutant, and a bitwise operation is
   // used to set the correct bit.
-  result << "            enabled_bitset[local_value / 64] |= (1 << "
+  result << "            enabled_bitset[local_value / 64] |= ((uint64_t) 1 << "
```

Comment:
I considered using `1LL` when fixing this issue. However, the "at least" part of the definition of `1LL` troubles me. Although no current implementation has `1LL` as a 128-bit type (and likely won't for a long time), the prelude function requires an exact 64-bit type to work. I chose `uint64_t` because it:
(a) specifies exactly 64 bits (not more or less), and
(b) clearly indicates to everyone that the type is 64 bits without needing to refer back to the C documentation.

Reply:
That makes sense - in that case please use the static_cast to uint64_t.